### PR TITLE
Additional Unit tests

### PR DESCRIPTION
**Pull request by bobloblaws93**
-Creation of additional unit tests

### CODE REVIEW

File: .sonar/report-task.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+projectKey=company:my-application
```
Comment (ghiobi) [inline .sonar/report-task.txt]: Please remove this file. `.gitingnore` it.

File: src/app/app.spec.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
-import { async, ComponentFixture, TestBed } from "@angular/core/testing";
-import { IonicModule } from "ionic-angular";
 import { TabsPage } from "../pages/tabs/tabs";
-import { MyApp } from "./app.component";
-
+import {async, ComponentFixture, TestBed} from "@angular/core/testing";
```
Comment (ghiobi) [inline src/app/app.spec.ts]: @ronnpang1 Spaces please.

File: src/pages/home/wallet-header/wallet-header.component.spec.ts [hunk 3]
```diff
@@ -2,28 +2,63 @@
 import { DebugElement } from "@angular/core";
 import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 import { By } from "@angular/platform-browser";
-
+import { CurrencyService } from "../../../providers/currency.service";
+import { NavController } from "ionic-angular";
```
Comment (ghiobi) [inline src/pages/home/wallet-header/wallet-header.component.spec.ts]: SPACES PLEASE.

File: src/pages/login/login.spec.ts [hunk 4]
```diff
@@ -0,0 +1,103 @@
+/**
+ * Created by ronniepang on 2017-03-19.
+ */
+import {Injectable} from "@angular/core";
+import {async, inject, TestBed,ComponentFixture} from "@angular/core/testing";
+import{DebugElement} from "@angular/core/";
+import {BaseRequestOptions, Http, HttpModule, Response, ResponseOptions} from "@angular/http";
```
Comment (ghiobi) [inline src/pages/login/login.spec.ts]: SPACES AGAIN.

File: src/pages/manage/deposit/deposit.component.spec.ts [hunk 5]
```diff
@@ -1,25 +1,84 @@
-import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import {async, ComponentFixture, TestBed} from "@angular/core/testing";
+import {Validators, FormBuilder, FormGroup} from "@angular/forms";
+import {NavParams,NavController} from "ionic-angular";
+import {DepositComponent} from "./deposit.component";
+import {Storage} from "@ionic/storage";
+import {BankTransferService} from "../../../providers/bank.service";
```
Comment (ghiobi) [inline src/pages/manage/deposit/deposit.component.spec.ts]: SPACES PLEASE.

File: src/pages/manage/withdraw/withdraw.component.spec.ts [hunk 6]
```diff
@@ -1,25 +1,79 @@
-import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import {async, ComponentFixture, TestBed} from "@angular/core/testing";
+import {Validators, FormBuilder, FormGroup} from "@angular/forms";
+import {NavParams} from "ionic-angular";
```
Comment (ghiobi) [inline src/pages/manage/withdraw/withdraw.component.spec.ts]: SPACES PLEASE.

File: src/pages/send-money/send-money.spec.ts [hunk 7]
```diff
@@ -0,0 +1,104 @@
+/**
+ * Created by ronniepang on 2017-03-19.
```
Comment (ghiobi) [inline src/pages/send-money/send-money.spec.ts]: REMOVE THIS.

File: src/pages/send-money/send-money.spec.ts [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+ */
+/**
+ * Created by ronniepang on 2017-03-19.
+ */
+import {async, ComponentFixture, TestBed} from "@angular/core/testing";
+import {Validators, FormBuilder, FormGroup} from "@angular/forms";
+import {NavParams,AlertController, Loading, LoadingController,ModalController,NavController} from "ionic-angular";
```
Comment (ghiobi) [inline src/pages/send-money/send-money.spec.ts]: SPACES PLEASE.

File: src/pages/transfers/transfers.spec.ts [hunk 9]
```diff
@@ -0,0 +1,92 @@
+/**
+ * Created by ronniepang on 2017-03-19.
+ */
+import {async, ComponentFixture, TestBed} from "@angular/core/testing";
+import {Validators, FormBuilder, FormGroup} from "@angular/forms";
+import {NavParams,AlertController, Loading, LoadingController,ModalController,NavController} from "ionic-angular";
```
Comment (ghiobi) [inline src/pages/transfers/transfers.spec.ts]: SPACES PLEASE.

File: src/pages/transfers/transfers.spec.ts [hunk 10]
```diff
@@ -0,0 +1,92 @@
+/**
+ * Created by ronniepang on 2017-03-19.
```
Comment (ghiobi) [inline src/pages/transfers/transfers.spec.ts]: REMOVE THIS.

**Review by ghiobi — CHANGES_REQUESTED** [review]
@ronnpang1 Please correct these issues by tonight 8:00 EST.

**Comment by JTalarico** [comment]
"-Creation of additional unit tests"

Next time please specify which unit tests have been added. This is very vague.

File: src/pages/manage/deposit/deposit.component.spec.ts [hunk 11]
```diff
@@ -1,25 +1,84 @@
...
+import {MyApp} from "../../../app/app.component";
```
Comment (Scieon) [inline src/pages/manage/deposit/deposit.component.spec.ts]: Please refer to comment about spaces please

File: src/providers/auth.service.ts [hunk 12]
```diff
@@ -87,15 +87,15 @@ export class AuthService {
    * @param unlock
    * @returns {Observable|"../../Observable".Observable|"../../../Observable".Observable}
    */
-  public otp(data, unlock){
+  public otp(data, unlock) {
     let response = new Observable((observer) => {
-      this.authHttp.post(environment.server_url + "/api/auth/otp" + ((unlock)? "/unlock" : ""), data)
+      this.authHttp.post(environment.server_url + "/api/auth/otp" + ((unlock) ? "/unlock" : ""), data)
```
Comment (JTalarico) [inline src/providers/auth.service.ts]: Why create a variable for response? Why not just return the Observable immediately?

**Review by ghiobi — APPROVED** [review]

**Review by ghiobi — CHANGES_REQUESTED** [review]
Please remove your generated comment.

**Review by ghiobi — APPROVED** [review]

**Pull request merged**